packages/ember-glimmer/lib/helpers/component.js

```diff
@@ -206,6 +205,11 @@ function createCurriedDefinition(definition, args) {
   );
 }
 
+const EMPTY_BLOCKS = {
```

Comment:
at least in dev mode, freezing it does make it a different "shape" polymorphic with the unfrozen versions, not sure if it is already polymorphic and empty and non empty should be both seen by the time it is JITed so may not matter.